### DISCUSSION TITLE
Introducing cva.style

### DISCUSSION THREAD

**Discussion by joe-bell** [joe-bell/cva#117] [Announcements]
# Introducing [cva.style](https://cva.style)

The new-and-improved documentation for `cva` ✨

**Comment by Sam-Apostel** [comment]
Have you considered an alternative for stackblitz on unsupported browsers?

**Reply by joe-bell** [reply]
I’ll be sticking with StackBlitz for now as it’s superior in performance compared to alternatives like CodeSandbox. A fallback link to GitHub could be fine in the meantime?

**Reply by Sam-Apostel** [reply]
An alternative could just be a code block with the example. GitHub link might be good enough.

Accepting PR's for this?

**Reply by joe-bell** [reply]
Not at the moment, thanks though

**Reply by joe-bell** [reply]
Fixed in #119
